### PR TITLE
(maint) fix typos

### PR DESCRIPTION
**Pull request by ehom**
clean up for localization-friendliness

### CODE REVIEW

**Comment by gguillotte** [comment]
@ehom Looks fine to me from a support perspective but I'll leave docs reviews to @jbondpdx.

**Review by jbondpdx — CHANGES_REQUESTED** [review]
Looks good to me. One minor change and then good to go!

File: README.md [hunk 1]
```diff
@@ -318,16 +318,16 @@ azure_vm { 'ssd-example':
...
+The Azure portal provides switches to enable _boot_diagnostics_ and _guest diagnostics_. Both switches require access to a storage account to dump the diagnostic data.
 
 The switch behaves differently depending what is activated:
 
-* Boot diagnostics - Configures the VM `diagnosticsProfile` setting to write out boot diagnostics .  If required, manually enable using the portal.  Since boot diagnostics only apply at boot time, their most useful for interactive debugging when a VM is having a problems booting.  If required, boot diagnostics can be enabled through the Azure portal.
-* Guest diagnostics - Configures an extension to capture live diagnostic output.  This needs to be _different_ depending on the selected guest OS and is enabled by supplying the appropriate data to the `extensions` parameter.
+* Boot diagnostics - Configures the VM `diagnosticsProfile` setting to write out boot diagnostics. If required, manually enable using the portal. Since boot diagnostics only apply at boot time, their most useful for interactive debugging when a VM is having a problems booting. If required, boot diagnostics can be enabled through the Azure portal.
```
[the remark this answers is not in the document]
Comment (ehom) [reply README.md]: Found one more:

> their most useful

they're most useful

**Review by clairecadman — APPROVED** [review]
I fixed the two typos on line 329

**Pull request closed without merging**